src/libappimage/desktop_integration/integrator/Integrator.cpp

```diff
@@ -192,17 +216,40 @@ namespace appimage {
                     }
                 }
 
+                void deployMimeTypeIcons() {
+                    std::list<std::string> mimeTypeIconNames;
+                    for (const auto& editor : mimeInfoFiles) {
```

Comment:
`std::for_each` for all three loops perhaps?

Reply:
Why? Don't see any benefit on using it.

Reply:
The code gets a little more compact. But I don't insist on it.